README.md

```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
 Groups
 ------
 
-When using [ACBS](https://github.com/AOSC-Dev/acbs/) (Autobuild CI Build System),
+When using [ACBS](https://github.com/AOSC-Dev/acbs) (Autobuild CI Build System),
 it is possible to build a series of packages with a single command. This is
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
-Automation
-----------
+Notes for developers on Windows
+-------------------------------
 
-Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
+Currently, this repository cannot be checked out on Windows, since there're
+backslashes in file names, specifically required by [systemd.mount](https://www.freedesktop.org/software/systemd/man/latest/systemd.mount.html#Description)
+for [string escaping](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#String%20Escaping%20for%20Inclusion%20in%20Unit%20Names),
+which is [not allowed by Windows](https://learn.microsoft.com/windows/win32/fileio/naming-a-file#naming-conventions).
+Notebly, using MinGW environments to checkout the repository will fail either.
```

Comment:
Notably, repository checkout will also fail in MinGW environments (because?)

Reply:
The reason is that MinGW still works in Win32 subsystem, but that's out of scope of this document. I'm working on verifying if Cygwin will checkout the repository.

Reply:
> The reason is that MinGW still works in Win32 subsystem, but that's out of scope of this document. I'm working on verifying if Cygwin will checkout the repository.

Nice, further clarification would be best included

Reply:
Cygwin doesn't help either.